types/yargs-unparser/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for yargs-unparser 2.0
+// Project: https://github.com/yargs/yargs-unparser#readme
+// Definitions by: Anton Golub <https://github.com/antongolub>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```

Comment:
```suggestion
```
It's already unsupported on DT, 
let me know if you've used kind of template to create types (so TS 2.2 was added), maybe it needs an update, thx!